optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/bavet/uni/BavetAbstractUniConstraintStream.java

```diff
@@ -63,6 +63,11 @@ public void addChildStream(BavetAbstractUniConstraintStream<Solution_, A> childS
         childStreamList.add(childStream);
     }
 
+    @Override
+    public int getCardinality() {
+        return 1;
```

Comment:
I am ok with doing it like this, but we could just as well do it as a default method on UniConstraintStream. Either way, it's all good.